src/symbol/glyph_source.js

```diff
@@ -50,22 +55,33 @@ class GlyphSource {
         // the number of pixels the sdf bitmaps are padded by
         const buffer = 3;
 
-        const missing = {};
+        const missingRanges = {};
         let remaining = 0;
 
         const getGlyph = (glyphID) => {
             const range = Math.floor(glyphID / 256);
+            if (this.localIdeographFontFamily &&
+                (isChar['CJK Unified Ideographs'](glyphID) ||
+                 isChar['Hangul Syllables'](glyphID))) {
```

Comment:
Any of the CJK Unified Ideographs Extension blocks would be good candidates. We don’t support them at all right now, yet they could theoretically appear in OpenStreetMap data. Overpass turbo doesn’t seem to be capable of searching for supplementary characters in character classes, but apparently rare characters do occur in Hong Kong place names: https://github.com/mapbox/DEPRECATED-mapbox-gl/issues/29#issuecomment-256296834.